Documentation/Books/Manual/Tutorials/Kubernetes/README.md

```diff
@@ -49,26 +49,36 @@ at cluster level.
 
 For now, any recent Kubernetes cluster will do (e.g. `minikube`).
 
-Then run (replace `<version>` with the version of the operator that you want to install):
+Then run:
```

Comment:
Oh dear, this is my mistake. The old text is the better one. We need to revert this particular change. I will revert in the original repo as well.